claims_db/phclaims/stage/tables/create_stage_mcare_mbsf_ab.yaml

```diff
@@ -0,0 +1,56 @@
+schema: stage
+table: mcare_mbsf_ab
+years:
+vars:
+    bene_id: VARCHAR(15) collate SQL_Latin1_General_Cp1_CS_AS
+    bene_enrollmt_ref_yr: SMALLINT
+    five_percent_flag: VARCHAR(1)
+    enhanced_five_percent_flag: VARCHAR(1)
```

Comment:
If these are all only ever going to be 1 character, you should change VARCHAR(1) to CHAR(1) as there is a space overhead from VARCHAR (needed to specify how many chars are included)

Reply:
changed varchar to char, as appropriate.